libraries/omnitruck_helpers.rb

```diff
@@ -0,0 +1,134 @@
+#
+# Copyright (c) 2014-2015, Chef Software, Inc. <legal@chef.io>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require_relative './helpers'
+require 'net/http'
+require 'json'
+
+module ChefIngredientCookbook
+  # The code in this module will eventually go into mixlib-install
+  module OmnitruckHelpers
+    include Helpers
+
+    # TODO: This method should get product name as a parameter and find the
+    # latest version for it.
+    def current_version(product_name)
+      unless %w(chef chefdk).include?(product_name)
+        fail "Unknown product #{product_name}"
+      end
+
+      JSON.parse("/opt/#{product_name}/version-manifest.json")['build_version']
```

Comment:
Are the products that have `version-manifest.json` consistently installed in `/opt/PRODUCT_NAME` now? I know we used to have `/opt/opscode` for Enterprise Chef and Private Chef.